systems/plants/viewer/drake_urdf_renderer.cpp

```diff
@@ -345,7 +345,7 @@ static void handle_lcm_viewer_load_robot(const lcm_recv_buf_t *rbuf, const char
   drake_lcmt_viewer_command_publish(self->lcm, "DRAKE_VIEWER_STATUS", &status_message);
   cout << "successfully loaded model" << endl;
 
-  bot_viewer_request_redraw(self->viewer);
+  // bot_viewer_request_redraw(self->viewer);
```

Comment:
We had a draw command on load, but it does not have a known posture, so always drew at q=0 .  That was bad for the new test case.  Removing it did not seem to have any negative impact when i ran a bunch of the ither examples.  I think it's better out



> On Jul 7, 2015, at 5:23 PM, John Carter <notifications@github.com> wrote:
> 
> In systems/plants/viewer/drake_urdf_renderer.cpp:
> 
> > @@ -345,7 +345,7 @@ static void handle_lcm_viewer_load_robot(const lcm_recv_buf_t *rbuf, const char
> >    drake_lcmt_viewer_command_publish(self->lcm, "DRAKE_VIEWER_STATUS", &status_message);
> >    cout << "successfully loaded model" << endl;
> >  
> > -  bot_viewer_request_redraw(self->viewer);
> > +  // bot_viewer_request_redraw(self->viewer);
> Was this not needed?
> 
> —
> Reply to this email directly or view it on GitHub.
>